lib/email/bounces.js

```diff
@@ -52,56 +45,66 @@ module.exports = function (log, error) {
 
     function handleBounce(message) {
       var recipients = []
-      if (message.bounce && message.bounce.bounceType === 'Permanent') {
```

Comment:
We want to record **all** bounces and complaints, before this we were only reporting hard bounces and 'abuse' complaints.

Writing this comment made me wonder, would we want to delete unverified accounts because of soft bounces? If not, I could add a check for `'Permanent'` before `deleteAccountIfUnverified`.